packages/@okta/vuepress-theme-prose/components/SidebarItem.vue

```diff
@@ -66,11 +74,22 @@ export default {
       hidden: !!this.link.hidden
     };
   },
+  computed:{
+    isLink: function(){
+      return this.link.hasOwnProperty('path') && this.link.path !== null
+    },
+    isParent: function(){
+      return !this.link.hasOwnProperty('path') && this.link.hasOwnProperty('subLinks')
+    },
+    isBlankDivider: function(){
```

Comment:
`isLink` and `isBlankDivider` can it be combined into one?

Reply:
No. We have new entity here.